src/main/python/keyboard_comm.py

```diff
@@ -222,6 +228,15 @@ def reload_layout(self, sideload_json=None):
         self.rows = payload["matrix"]["rows"]
         self.cols = payload["matrix"]["cols"]
 
+        if "customKeycodes" in payload:
+            custom_keycodes = payload["customKeycodes"]
+            self.custom_keycodes = []
```

Comment:
As mentioned above, this could simply be `self.custom_keycodes = payload["customKeycodes"]`, or even removing the `if`: `self.custom_keycodes = payload.get("customKeycodes")`